astropy/nddata/tests/test_ccddata.py

```diff
@@ -1021,3 +1021,13 @@ def test_read_returns_image(tmpdir):
     ccd = CCDData.read(filename, unit='adu')
     # Expecting to get (5, 5), the size of the image
     assert ccd.data.shape == (5, 5)
+
+
+# https://github.com/astropy/astropy/issues/9664
```

Comment:
Nitpicking: Wouldn't it make more sense to add this as docstring for the test-function instead of as leading comment.

But feel free to ignore this. 😄

Reply:
I copied the style of the test above fwiw.

Reply:
okay, then let's be consistent :)